Fix registration of multiple callbacks when using the global meter provider (#4945)
open-telemetry/opentelemetry-go: CHANGELOG.md

- Add exemplar support to `go.opentelemetry.io/otel/exporters/otlp/otlpmetric/otlpmetricgrpc`. (#4900)
- Add exemplar support to `go.opentelemetry.io/otel/exporters/otlp/otlpmetric/otlpmetrichttp`. (#4900)

### Fixed

> - Fix registration callback missing called when register multiple callbacks to `go.opentelemetry.io/internal/global`. (#4945)

Reviewer: ```suggestion
- Fix registration of multiple callbacks in `go.opentelemetry.io/internal/global`. (#4945)
```

Reviewer 2: `go.opentelemetry.io/internal/global` is not a "public" package. Maybe something more or less like

```suggestion
- Fix registration callback missing called when registering multiple callbacks when using the global meter provider from `go.opentelemetry.io/otel`. (#4945)
```

(I do not like this sentence, but I hope it captures my intention)
